updated start command for avahi-daemon to work with Debian 12 (fixes …

z1r0- opened the pull request:
This PR is a fix for #384

buanet commented:
Hi,
thank you for this pr.
Seems to work. But pls add `>> /opt/scripts/avahi_startup.log 2>&1` for log/ error output. :)
Then I will create a new beta for you to test. 

Regards, 
André

z1r0- commented:
@buanet I skipped it because -D also redirects the log to syslog and we would just end up with an empty file. I changed it to start without -D so we can write the output to /opt/scripts/avahi_startup.log and added an additional `&` at the end to put it in the background.
